### PR TITLE
chore: add prefix to identify gitlink command in command palette

### PR DESCRIPTION
**Pull request by xiaohanxu-nick**
as title

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,11 +34,11 @@
     "commands": [
       {
         "command": "extension.gotoOnlineLink",
-        "title": "Goto Online Link"
+        "title": "Gitlink: Goto Online Link"
```
Comment (qinezh) [inline package.json]: Would you please update to `GitLink`? Thanks~

Comment (xiaohanxu-nick) [reply package.json]: sure

**Review by qinezh — APPROVED** [review]
LGTM :shipit:

**Comment by qinezh** [comment]
Thank you for your contribution. @Xiaohanxu1996

**Pull request merged**